scripts/wrappers/tests/test-cluster-health-green.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -eu
+
+usage() {
+cat << EOF
+usage: test-cluster-health-green.sh --admin-auth-password admin
+Tests if the cluster's health status is green.
+--admin-auth-password  (Optional) Password for basic auth with the opensearch rest api, default "admin"
+--help                            Shows help menu
+EOF
+}
+
+
+# Args
+admin_auth_password=""
+
+
+# Args handling
+function parse_args () {
+    # init-security boolean - from the charm, this should be based on a flag on the app data bag.
+    local LONG_OPTS_LIST=(
+        "admin-auth-password"
+        "help"
+    )
+    local opts=$(getopt \
+      --longoptions "$(printf "%s:," "${LONG_OPTS_LIST[@]}")" \
+      --name "$(readlink -f "${BASH_SOURCE}")" \
+      --options "" \
+      -- "$@"
+    )
+    eval set -- "${opts}"
+
+    while [ $# -gt 0 ]; do
+        case $1 in
+            --admin-auth-password) shift
+                admin_auth_password=$1
+                ;;
+            --help) usage
+                exit
+                ;;
+        esac
+        shift
+    done
+}
+
+function set_defaults () {
+    if [ -z "${admin_auth_password}" ]; then
+        admin_auth_password="admin"
+    fi
+}
+
+
+parse_args "$@"
+set_defaults
+
+
+# Check cluster health
+endpoint="https://localhost:9200/_cluster/health"
+
+health_resp=$(curl -k -XGET "${endpoint}" -u "admin:${admin_auth_password}")
+echo -e "Cluster Health Response: \n ${health_resp}"
+
+cluster_status=$(echo "${health_resp}" | yq -r .status)
```

Comment:
`yq` isn't installed by default on most distros I don't think. Is it listed as a dep somewhere?

```bash
~  ❯ lxc launch -p default ubuntu:22.04 ubuntu-22
Creating ubuntu-22
Starting ubuntu-22                        

~  ❯ lxc exec ubuntu-22 -- bash                  
root@ubuntu-22:~# yq
yq: command not found
root@ubuntu-22:~# exit
exit
Error: Command not found

~  ❯ lxc launch -p default ubuntu:20.04 ubuntu-20
Creating ubuntu-20
Starting ubuntu-20                        

~  ❯ lxc exec ubuntu-20 -- bash                  
root@ubuntu-20:~# yq

Command 'yq' not found, but can be installed with:

snap install yq

root@ubuntu-20:~# exit
exit

~  ❯ 
```

Reply:
You're right, it must must be installed by us, it's part of the list of snaps I install during the staging process, you can find it [here](https://github.com/canonical/opensearch-snap/blob/edge/snap/snapcraft.yaml#L104-L105)

Reply:
Great, no issue then!